src/site/_includes/layout.njk

```diff
@@ -13,11 +13,13 @@
       <meta name="offline" content="true" />
     {%- endif %}
 
-    {% if not nometa %}
+    {% if nometa %}
+      {# Area on page that service worker will load title, rss url and if offline #}
+      <title>web.dev</title>
```

Comment:
How about using the title from `site.js` here?
Could also add a comment to `site.js` explaining that changing the title in that file will affect the service worker.